include/deal.II/fe/fe.h

```diff
@@ -758,7 +758,18 @@ class FiniteElement : public Subscriptor,
    * Virtual destructor. Makes sure that pointers to this class are deleted
    * properly.
    */
-  virtual ~FiniteElement ();
+  virtual ~FiniteElement () = default;
+
+  /**
+   * Creates information for creating a FESystem with this class as
+   * base element and with multiplicity @p multiplicity. In particular,
+   * the return type of this function can be used in the constructor
+   * for a FESystem object.
+   * This function calls clone() and hence creates a copy of the
+   * current object.
+   */
+  std::pair<std::unique_ptr<FiniteElement<dim, spacedim> >, unsigned int>
+  operator^ (unsigned int multiplicity);
```

Comment:
can you make the argument `const` as we typically do for input arguments?

Reply:
Yes. It's always a bit weird to call `std::move` on a `const` object but here it doesn't really matter.

Reply:
Oh, yes, you can always make the whole *function* `const`, but I really just meant the `unsigned int` argument.

Reply:
Turned out I was wrong. `std::make_pair` requires `unsigned int &&` and `std::move(const unsigned int)` just gives me `const unsigned int`. However, making the function `const` works.